client/driver/executor/executor_windows.go

```diff
@@ -56,14 +56,3 @@ func sendCtrlBreak(pid int) error {
 	}
 	return nil
 }
-
-// Send the process a Ctrl-Break event, allowing it to shutdown by itself
-// before being Terminate.
-func (e *UniversalExecutor) shutdownProcess(proc *os.Process) error {
-	if err := sendCtrlBreak(proc.Pid); err != nil {
```

Comment:
Ah this is from removing the shutdown 😕 We will likely have to keep it so that we have a spot to do special shutdown handling that isn't just signaling.